### ISSUE TITLE
Timing of next point release?

### ISSUE THREAD

**Issue by gkz** [prettier/prettier#8776]
Hi, thank you for being such active maintainers of Prettier! PRs get reviewed very quickly.

I was just wondering if there is a timeline for the next point release (2.0.6).

Thanks!

**Comment by thorn0** [comment]
There is a bunch of things in `master` that need to be finished before releasing 2.1. If some specific fix urgently needs to be released, we can cherry-pick it into the `patch-release` branch and release it from there. That's how 2.0.5 was done.

**Comment by gkz** [comment]
Ah OK, so master is tracking a future 2.1 release rather than a 2.0.6 release, makes sense. I recently had #8768 merged and was just wondering what the approximate time frame would be for it being released. It's not super urgent.

**Comment by bakkot** [comment]
@thorn0 which things in master need to be finished before releasing 2.1? Is there a reason it can't be released now, so that features like https://github.com/prettier/prettier/pull/7875 can go out, and whatever those other things are be pushed to 2.2?

**Comment by thorn0** [comment]
@bakkot 

* Finalize the update of remark: #8630
* Printing of member chains: #7889 has been merged, and I started but didn't finish #8063. I expect that if we release one without the other, there will be a lot of criticism again as it happened after 2.0, but together they might be good enough.

**Comment by bakkot** [comment]
@thorn0 If those PRs weren't good enough to ship, shouldn't they be backed out and re-landed once they're done? I'd be happy to do that if you'd like.

**Comment by thorn0** [comment]
@bakkot I'm fine with reverting #7889 but not remark. That update is really long-overdue and fixes too many issues.

**Comment by justingrant** [comment]
I've been eagerly waiting for #4216 (the fix to #4926) which I'm hoping will fix the issue with VSCode randomly indenting every line in a class method when I paste one line of code into it.  If you do end up doing a point release instead of 2.1, it'd be great to see that fix in it. 

P.S. - Prettier is wonderful. Thanks so much for maintaining it!

**Comment by sosukesuzuki** [comment]
/cc @thorn0 
We've finished #8630 so we can revert #7889 and work for releasing 2.1.

**Comment by sosukesuzuki** [comment]
@fisker @evilebottnawi @thorn0 What about other todos in [2.1 milestone](https://github.com/prettier/prettier/milestone/35)? Should we postpone implementing them until 2.2?  I'm not sure about status.

**Comment by evilebottnawi** [comment]
Agree, we can postpone other changes to `2.2` and do release `2.1` (markdown fixes)

**Comment by sosukesuzuki** [comment]
Okay I've created PR reverts 7889. And I'll remove other changes from 2.1 milestone.

**Comment by sosukesuzuki** [comment]
I opened 2.1 checklist issue #8882

**Comment by sosukesuzuki** [comment]
We've already released 2.1:+1:
